tests/outputs/CMakeLists.txt

```diff
@@ -15,3 +15,6 @@ target_link_libraries(TestDyd DynaFlowLauncher::outputs)
 
 DEFINE_TEST(TestPar OUTPUTS)
 target_link_libraries(TestPar DynaFlowLauncher::outputs)
+
+DEFINE_TEST(TestDiagram OUTPUTS)
+target_link_libraries(TestDiagram DynaFlowLauncher::outputs)
```

Comment:
Missing new line

Reply:
Not a requirement for cmakelists, it is the case for other cmakelists